router.estimateRouteFee : panic: runtime error: invalid memory address or nil pointer dereference

running a test nodejs app calling ldn via GRPC

    let dest = Buffer.from('node_pubkey', 'hex');
    let queryRoutesReqRouter = { dest: dest, amt_sat: decoded.num_satoshis};
    router.estimateRouteFee(queryRoutesReqRouter, function (err, response) {
        // ... code
    });

lnd crashes with error 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xbf75c8]

std terminal error
https://www.dropbox.com/s/54uhwkp4d4aphm9/std_out.log?dl=1

Log with debug = trace
https://www.dropbox.com/s/992cjmmuntlgsd2/lnd.zip?dl=1


### Your environment

v0.11.0-beta
neutrino
Linux  4.15.0-112-generic #113-Ubuntu  x86_64 x86_64 x86_64 GNU/Linux

--- a/lnrpc/routerrpc/router_server.go
+++ b/lnrpc/routerrpc/router_server.go
@@ -320,11 +320,13 @@ func (s *Server) EstimateRouteFee(ctx context.Context,
 	// that target amount, we'll only request a single route. Set a
 	// restriction for the default CLTV limit, otherwise we can find a route
 	// that exceeds it and is useless to us.
+	mc := s.cfg.RouterBackend.MissionControl
 	route, err := s.cfg.Router.FindRoute(
 		s.cfg.RouterBackend.SelfNode, destNode, amtMsat,
 		&routing.RestrictParams{
-			FeeLimit:  feeLimit,
-			CltvLimit: s.cfg.RouterBackend.MaxTotalTimelock,
+			FeeLimit:          feeLimit,
+			CltvLimit:         s.cfg.RouterBackend.MaxTotalTimelock,
+			ProbabilitySource: mc.GetProbability,
 		}, nil, nil, s.cfg.RouterBackend.DefaultFinalCltvDelta,
 	)
 	if err != nil {

